### PR TITLE
Bulk Validation: fix missing descriptions

### PR DESCRIPTION
**Pull request by jmirabal80**

### CODE REVIEW

File: iis/extensions/smooth-streaming-client/chunkinfo-class-microsoft-web-media-smoothstreaming.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: ChunkInfo Class (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: ChunkInfo Class
+description: Describes the ChunkInfo Class and provides the field's namespace, assembly, syntax, properties, methods and version information.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/chunkinfo-class-microsoft-web-media-smoothstreaming.md:4]: Describes the ChunkInfo Class and provides the field's namespace, assembly, syntax, properties, methods and version information.
Suggestion: Describes the ChunkInfo class

File: iis/extensions/smooth-streaming-client/clipinformation-clickthroughuri-property-microsoft-web-media-smoothstreaming_1.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: ClipInformation.ClickThroughUri Property (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: ClickThroughUri Property
+description: Describes the ClickThroughUri Property and provides the namespace, assembly, syntax, property value, and version information.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/clipinformation-clickthroughuri-property-microsoft-web-media-smoothstreaming_1.md:4]: Describes the ClickThroughUri property

File: iis/extensions/smooth-streaming-client/clipinformation-usesmoothstreamingcache-property-microsoft-web-media-smoothstreaming_1.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: ClipInformation.UseSmoothStreamingCache Property (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: UseSmoothStreamingCache Property
+description: Describes the UseSmoothStreamingCache Property and provides the namespace, assembly, syntax, and version information.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/clipinformation-usesmoothstreamingcache-property-microsoft-web-media-smoothstreaming_1.md:4]: Describes the UseSmoothStreamingCache property

File: iis/extensions/smooth-streaming-client/smoothstreamingexception-class-microsoft-web-media-smoothstreaming.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: SmoothStreamingException Class (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: SmoothStreamingException Class
+description: This article contains namespace, assembly, syntax, properties, methods, and version information for the SmoothStreamingException Class.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/smoothstreamingexception-class-microsoft-web-media-smoothstreaming.md:4]: SmoothStreamingException class.

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-liveplaybackoffset-property-microsoft-web-media-smoothstreaming_1.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: SmoothStreamingMediaElement.LivePlaybackOffset Property (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: LivePlaybackOffset Property
+description: This article contains syntax, property value, and version information for the SmoothStreamingMediaElement.LivePlaybackOffset Property.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-liveplaybackoffset-property-microsoft-web-media-smoothstreaming_1.md:4]: SmoothStreamingMediaElement.LivePlaybackOffset property.

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-scheduleclip-method-clipinformation-clipcontext-object-microsoft-web-media-smoothstreaming_1.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 title: SmoothStreamingMediaElement.ScheduleClip Method (ClipInformation, ClipContext, Object) (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: ScheduleClip Method (ClipInformation, ClipContext, Object)
+description: Describes the ScheduleClip Method (ClipInformation, ClipContext, Object) and provides the namespace, assembly, syntax, parameters, and version information.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-scheduleclip-method-clipinformation-clipcontext-object-microsoft-web-media-smoothstreaming_1.md:4]: ScheduleClip methods

File: iis/extensions/smooth-streaming-client/streaminfo-restricttracks-method-microsoft-web-media-smoothstreaming_1.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 title: StreamInfo.RestrictTracks Method  (Microsoft.Web.Media.SmoothStreaming)
 TOCTitle: RestrictTracks Method
+description: Describes the StreamInfo.RestrictTracks Method and provides the namespace, assembly, syntax, parameters, and version information.
```
Comment (MeeraDi) [inline iis/extensions/smooth-streaming-client/streaminfo-restricttracks-method-microsoft-web-media-smoothstreaming_1.md:4]: StreamInfo.RestrictTracks method

**Comment by jmirabal80** [comment]
**Global effort to fix build validation errors**

@Rick-Anderson and @mcleblanc 

The warnings will be addressed in another PR!

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**